server/client_integration_test.go

```
@@ -466,7 +466,12 @@ func testRetrieveWithNixCopy(ctx context.Context, t *testing.T, testService *ser
 
 	// Configure the binary cache URL using the same format as Nix's own tests
 	endpoint := testService.MinioClient.EndpointURL().Host
-	binaryCacheURL := fmt.Sprintf("s3://%s?endpoint=http://%s&region=eu-west-1", testService.Bucket, endpoint)
+	storeDir := os.Getenv("NIX_STORE_DIR")
```

golangci-lint (2.5.0)

[error] missing whitespace above this line (too many statements above if)

(wsl_v5)